strings/c-or-cpp/Permutation of String.c

```diff
@@ -0,0 +1,57 @@
+#include<stdio.h>
+//declare permutation and swap functions
+void permutation(char *,int,int);
+void swap(char *,char *);
+main()
+{
+    char s[10];
+	//taking string
+    printf("enter string");
+    scanf("%s",&s);
+	//calling permutation
+    permutation(&s,0,strlen(s)-1);
```

Comment:
passing reference of the pointer to the string which will make the argument received as pointer to pointer which will give out undesired result at runtime as the function doesn't treat it like pointer to pointer.

Reply:
can you please explain it more clear i did understand
please